gitax/Visual Studio Projects/C Sharp Projects/GTXLibGit2Sharp/GTXLibGit2Sharp.csproj

```diff
@@ -84,17 +85,33 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <AXProxy Include="Class.Global.axproxy">
```

Comment:
What's the other way of reporting errors to the user from the C# code? Are you proposing to return a particular error message/code and let the X++ code handle it? I'm fine with that. The reason I added it was because there was nothing in the UI that gave me the real underlying issue. Only when I debugged did I notice the error.